Src/Plugins/PlaintextSecrets/PlaintextSecrets.json

```diff
@@ -10,7 +10,7 @@
       "MatchExpressions": [
         {
           "SubId": "AzureDevOpsPersonalAccessToken",
-          "Base64DecodedContentLength": 70,
+          "MatchLengthToDecode": 52,
```

Comment:
>MatchLengthToDecode [](start = 11, length = 19)

Note this change! We now specify the length of the non-encoded value (as this is a known value). We then *compute* the encoded length from this. Changed the property name to reflect the difference in meaning.